### PR TITLE
add settings import/export to fontSettings

### PR DESCRIPTION
**Pull request by patrickkettner**
The advanced font settings is the repo for [Advanced Font Settings](https://chrome.google.com/webstore/detail/advanced-font-settings/caclkomlalccbpcdllchkeecicepbmbm) repo. A developer reached out to request this functionality, so I am updating the sample for approval before publishing it.

### CODE REVIEW

File: api-samples/fontSettings/fontSettings Advanced/options.js [hunk 1]
```diff
@@ -593,6 +599,177 @@ advancedFonts.closeOverlay = function () {
...
+    });
+  }
+  if (isNumeric('defaultFontSize', config)) {
+    await chrome.fontSettings.setDefaultFontSize({
+      pixelSize: config.minimumFontSize
+    });
+  }
+
+  config.configuredFonts.forEach(({ script, scriptData }) => {
+    scriptData.forEach(async ({ fontId, genericFamily }) => {
```
Comment (oliverdunk) [inline api-samples/fontSettings/fontSettings Advanced/options.js:678]: We should probably handle bad data here, e.g `configuredFonts` where `scriptData` does not exist or is not an array?

File: api-samples/fontSettings/fontSettings Advanced/options.js [hunk 2]
```diff
@@ -593,6 +599,177 @@ advancedFonts.closeOverlay = function () {
...
+      await chrome.fontSettings.setFont({
+        fontId,
+        genericFamily,
```
Comment (oliverdunk) [inline api-samples/fontSettings/fontSettings Advanced/options.js:681]: We should probably handle bad data here, e.g fontID or genericFamily does not exist or is not the right datatype?

**Review by oliverdunk — CHANGES_REQUESTED** [review]
Left a couple of comments but otherwise this seems reasonable, thanks for working on it :)

**Comment by patrickkettner** [comment]
@oliverdunk should be ready for a rereview

**Review by oliverdunk — APPROVED** [review]
Haven't had a chance to test unfortunately but otherwise LGTM.

**Pull request merged**